backend/src/main/java/hu/flowacademy/konyhatunder/service/RecipeService.java

```diff
@@ -40,6 +38,7 @@
     private final CategoryRepository categoryRepository;
     private final AmountOfIngredientRepository amountOfIngredientRepository;
     private final ImageStorageService imageStorageService;
+    private final IngredientRepository ingredientRepository;
```

Comment:
Itt csinalhatnatok constructort, amibne erteket adtok ezeknek, vagy @Autowired-el is injektalhattok, de alapvetoen bad practice-nek tartom hogy egy 3rd party dependencytol fuggjon hogy mukodik-e a dependency injection.

Reply:
+1, kifejezeten zavaró, én sem szeretem a Lombok-os megoldást, jobban örülnék a ténylegesen megírt konstruktornak az Autowired-es annotációval.

Reply:
Done.